src/Context/ProjectContext.tsx

```diff
@@ -0,0 +1,65 @@
+
+import { IContextProps } from "@/Interfaces";
+import baseUrl from "@/utils/Custom/Custom";
+import { createContext, useContext, useState } from "react";
+import { ToastContext } from "./ToastContext";
+
+
+interface ProjectContextTypes {
+  handleGetData: (type: string, id: any,pages:number) => any
+  tableData: any
+
+}
+
+export const ProjectContext = createContext<ProjectContextTypes | null>(null);
+
+
+
+const ProjectContextProvider = ({ children }: IContextProps) => {
+
+  const toast = useContext(ToastContext)
+  let headers = {
+    Authorization: `Bearer ${localStorage.getItem('adminToken')}`,
+  }
+
+  const [tableData, setTableData] = useState(null)
+
+  // const [Loading, setLoading] = useState(false)
+  const handleGetData = (type: string, id: any,pages:number) => {
+
+    if (type === "get" || type === "delete") {
+      return baseUrl[type](`/api/v1/Project/${id}`,{
+        headers,
+        params:{
+          pageSize:pages,
```

Comment:
the name pages here is misleading, as you mean the number of record or elements not the number of pages